### ISSUE TITLE
Loss after fine-tune vs loss when load fine-tuned checkpoint different

### ISSUE THREAD

**Issue by huutuongtu** [serp-ai/bark-with-voice-clone#59]
Thank you for this cool work. I am doing some experiments with fine-tuning on a different language (Vietnamese). I followed the notebook [train_semantic.ipynb](https://github.com/serp-ai/bark-with-voice-clone/blob/main/train_semantic.ipynb) to train text to semantic. First, I checked the default pretrained model, and the validation loss was 10.90. After training, the validation loss in the notebook was 2.59, and I had the latest checkpoint in **semantic_output/pytorch_model.bin**.  However, when I loaded the latest checkpoint to inference, the validation loss of text_to_semantic was now 6.35. When I inferred with [generate.ipynb](https://github.com/serp-ai/bark-with-voice-clone/blob/main/generate.ipynb), the quality of the audio was so poor, and it did not correspond to the text. Am I doing something wrong?

**Comment by devinschumacher** [comment]
@huutuongtu 

We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉 Check it out: https://github.com/serpapps/ai-voice-cloner
